.luacheckrc

```diff
@@ -1,4 +1,4 @@
-std = "max"
+std = "min+sile"
 include_files = {
   "**/*.lua",
   "sile.in",
```

Comment:
```suggestion
  "sile.in",
```
This line looks like copy-paste cruft from SILE that is irrelevant to this project.